truetype/truetype_test.go

```diff
@@ -198,6 +198,52 @@ func TestIndex(t *testing.T) {
 	}
 }
 
+func TestName(t *testing.T) {
+	testCases := map[string]string{
+		"luxisr": "Luxi Sans",
+		"luximr": "Luxi Mono",
+		"luxirr": "Luxi Serif",
+	}
+
+	for name, want := range testCases {
+		f, testdataIsOptional, err := parseTestdataFont(name)
+		if err != nil {
+			if testdataIsOptional {
+				t.Log(err)
+			} else {
+				t.Fatal(err)
+			}
+			continue
+		}
+		if got := f.Name(NameIDFontFamily); got != want {
+			t.Errorf("%s: got %q, want %q", name, got, want)
+		}
+	}
+}
+
+func TestStyle(t *testing.T) {
```

Comment:
Delete this test now. It doesn't really add anything more than what TestName tests.